src/Storage/ABS/SourceFile.php

```diff
@@ -44,13 +46,15 @@ public function __construct(
         CsvOptions $csvOptions,
         bool $isSliced,
         array $columnsNames = [],
-        ?array $primaryKeysNames = null
+        ?array $primaryKeysNames = null,
+        ?string $blobMasterKey = null
     ) {
         parent::__construct($container, $filePath, $sasToken, $accountName);
         $this->isSliced = $isSliced;
         $this->csvOptions = $csvOptions;
         $this->columnsNames = $columnsNames;
         $this->primaryKeysNames = $primaryKeysNames;
+        $this->blobMasterKey = $blobMasterKey;
```

Comment:
co to je?

Reply:
to je potreba v TPT, tak to tam musim nejak dostat.